main.go

```diff
@@ -24,6 +24,8 @@ import (
 	// to ensure that exec-entrypoint and run can make use of them.
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
+	"github.com/medik8s/fence-agents-remediation/pkg/cli"
```

Comment:
Why to sort the imports?
Did you mean by using https://github.com/medik8s/node-healthcheck-operator/blob/main/Makefile#L156-L158 ?

Reply:
> Why to sort the imports?

because thet are unsorted?!

Reply:
I meant why do we care about sorting the imports? I don't mind of doing that
It is just new to me, and I just wonder since ATM in other operators we don't have it (e.g. NMO).

Reply:
We always sort imports. If you find unsorted imports somewhere, it was just missed in reviews

Reply:
Ok, then I will do that in a follow up PR.
What do you thing about moving https://github.com/slintes/sort-imports to be under [medik8s tools](https://github.com/medik8s/tools)?

Reply:
https://github.com/medik8s/fence-agents-remediation/pull/15

Reply:
it's not releated to medik8s, so I don't think that would be the right home for it